src/main/java/uk/gov/digital/ho/hocs/casework/domain/model/AbstractCaseData.java

```diff
@@ -100,10 +100,6 @@ public class AbstractCaseData implements Serializable {
     @Column(name = "date_received")
     private LocalDate dateReceived;
 
-    @Setter
-    @Getter
-    @Column(name = "completed")
-    private boolean completed;
```

Comment:
This is being removed from the top level DTO but there is no migration to remove it from the `case_data` table.

Reply:
I will include the completed column and will remove this as part of future migration (when we drop the column from case_data).